src/com/sap/piper/Notify.groovy

```diff
@@ -0,0 +1,51 @@
+package com.sap.piper
+
+import groovy.text.SimpleTemplateEngine
+
+class Notify implements Serializable {
+    private static enum Severity { ERROR, WARNING }
+    private final static String LIBRARY_NAME = 'piper-lib-os'
+    private final static String MESSAGE_PATTERN = '[${severity}] ${message} (${libName}/${stepName})'
+
+    protected static Utils instance = null
```

Comment:
naming: instance? What is instance? Normally this is used with singleton patterns - which we don't have here. What about `utils` (I'm sure there are much better names with a lower risk for mis-understanding)?

Reply:
right, renamed to `utils`